test/javascripts/acceptance/assigns-tab-user-menu-test.js

```diff
@@ -215,9 +215,9 @@ acceptance(
 );
 
 acceptance(
-  "Discourse Assign | experimental user menu | assign_enabled setting is disabled",
+  "Discourse Assign | user menu | assign_enabled setting is disabled",
   function (needs) {
-    needs.user({ redesigned_user_menu_enabled: true, can_assign: true });
```

Comment:
If we restore the `redesigned_user_menu_enabled; true`, will that allow us to merge this PR before merging the core change?